doc/build/html/user_guide/howto/visualize_model-1.py

```diff
@@ -0,0 +1,17 @@
+import tempfile
+import pathlib
+import ansys.acp.core as pyacp
+from ansys.acp.core.extras.example_helpers import ExampleKeys, get_example_file
```

Comment:
Ah, now I understood what you meant by moving `example_helpers` to a separate namespace. Are we envisioning anything else going into the `extras` namespace? 

If not, I think the extra nesting level is somewhat unnecessary.

Maybe the `mechanical_integration_helpers` from #673 could go there?

I'm also wondering about the `core` nesting level: At what point do we create an `ansys.acp.<something_else>`? I tend to think we'll only use that for "big" things outside of the PyACP core. For example, if we create a standalone utility for a specific task based around PyACP.

Reply:
My original idea of this extra layer was for tools and functions which are not used in a pyacp workflow but for testing, examples, documentation etc. This would be against adding `mechanical_integration_helpers` helpers there as well but that can be discussed. 

I thought about adding it under `ansys.acp.extras` but I decided against this option because it was not big enough. So I agree with you.

Reply:
Ok, yeah that makes sense.